coverage_comment/settings.py

```diff
@@ -27,6 +29,7 @@ class Config:
     GITHUB_REF: str
     GITHUB_EVENT_NAME: str
     GITHUB_PR_RUN_ID: int | None
+    COMMENT_TEMPLATE: str = template.read_template_file()
```

Comment:
I can change that, but that would break all tests that don't specify `COMMENT_TEMPLATE`, because then the clean method is never called. (When I run it locally in a Docker container, 4 tests fail because of this change).
Should I change it anyway and rewrite the tests to read the template from the file and pass it to the configuration?